### ISSUE TITLE
Docs: better documentation on struct tags

### ISSUE THREAD

**Issue by dcormier** [alecthomas/jsonschema#68]
There is no comprehensive docs on available struct tags, leaving users to have to dive into the code to figure out how to generate the schema they want. All the exists are some examples in the readme. It would be nice if there was a comprehensive guide in the package godoc.

**Comment by alecthomas** [comment]
Contributions are welcome.

**Comment by jackmahoney** [comment]
I agree on this issue, but thanks for making such a great package. Will try add prs if I get time. Keep it up
